qdrant_client/client_base.py

```diff
@@ -106,6 +106,89 @@ def recommend_groups(
     ) -> types.GroupsResult:
         raise NotImplementedError()
 
+    def discover(
+        self,
+        collection_name: str,
+        target: Optional[types.TargetVector] = None,
+        context: Optional[Sequence[types.ContextExamplePair]] = None,
+        query_filter: Optional[types.Filter] = None,
+        search_params: Optional[types.SearchParams] = None,
+        limit: int = 10,
+        offset: int = 0,
+        with_payload: Union[bool, List[str], types.PayloadSelector] = True,
+        with_vectors: Union[bool, List[str]] = False,
+        using: Optional[str] = None,
+        lookup_from: Optional[types.LookupLocation] = None,
+        consistency: Optional[types.ReadConsistency] = None,
+        timeout: Optional[int] = None,
+        **kwargs: Any,
+    ) -> List[types.ScoredPoint]:
+        """
```

Comment:
Let's move the comment into `qdrant_client.py`, as all other docstrings

Reply:
I was thinking we should move all the docstrings into the base, wdyt?

Reply:
No, user initializes QdrantClient, comments should be there.